integration-tests/jiva/node-stickiness/replicadeletion/jiva_replica_deletion_test.go

```diff
@@ -0,0 +1,111 @@
+package replicadeletion
+
+import (
+	"fmt"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/openebs/maya/integration-tests/artifacts"
+	stickiness "github.com/openebs/maya/integration-tests/jiva/node-stickiness"
+	pod "github.com/openebs/maya/pkg/kubernetes/pod/v1alpha1"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	// auth plugins
+	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
+)
+
+const (
+	testTimes                            = 20
+	pvcArtifact artifacts.ArtifactSource = "../jiva_pvc_resource.yaml"
+	scArtifact  artifacts.ArtifactSource = "../jiva_sc_resource.yaml"
+)
+
+var _ = Describe("[jiva] [node-stickiness] jiva replica pod node-stickiness test", func() {
+	var (
+		// defaultReplicaLabel represents the jiva replica
+		defaultReplicaLabel = "openebs.io/replica=jiva-replica"
+		// defaultCtrlLabel represents the jiva controller
+		defaultCtrlLabel = "openebs.io/controller=jiva-controller"
+		// replicaLabel consist of defaultReplicaLabel and coressponding
+		// pvcLabel
+		replicaLabel string
+		// ctrlLabel consist of defaultReplicaLabel and coressponding
+		// pvcLabel
+		ctrlLabel                 string
+		jivaTestNamespace         string
+		podListObj                *v1.PodList
+		scInstaller, pvcInstaller *stickiness.TestInstaller
+	)
+	BeforeEach(func() {
+
+		By("Deploying jiva-single-replica storageclass")
+		scInstaller = stickiness.NewTestInstaller().
+			WithArtifact(scArtifact).
+			GetUnstructObj().
+			GetInstallerObj().
+			Install()
+
+		By("Deploying PVC using jiva-single-replica storageClass in jiva-test namespace")
+		pvcInstaller = stickiness.NewTestInstaller().
+			WithArtifact(pvcArtifact).
+			GetUnstructObj().
+			GetInstallerObj().
+			Install()
+
```

Comment:
Let us instantiate podKubeClient here !! & make use of this everywhere in this code.

Reply:
Done